projects/fw_102/src/main.c

```diff
@@ -0,0 +1,49 @@
+#include <stdbool.h>
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define ITERATION_TIMEOUT 0.5
```

Comment:
Usually we try not to use floating point numbers for our delays. Try defining it as 500 and using soft_timer_start_millis() instead

Reply:
Also, it's a good idea to add units to the define for improved readability